consensus/src/round_manager_test.rs

```diff
@@ -287,6 +287,15 @@ impl NodeSetup {
         ));
         let time_service = Arc::new(ClockTimeService::new(executor));
 
+        let window_size = onchain_consensus_config
+            .window_size()
+            .and_then(|window_size| {
+                if window_size <= usize::MAX as u64 {
```

Comment:
If we move all to `u64` it's obvious we don't need this check. (Which I prefer.)

Also it's not obvious we need this since it's just a test.